pkg/apply/common_test.go

```diff
@@ -125,7 +129,7 @@ func newTestInventory(
 ) inventory.Client {
 	// Use an Client with a fakeInfoHelper to allow generating Info
 	// objects that use the FakeRESTClient as the UnstructuredClient.
-	invClient, err := inventory.ClusterClientFactory{StatusPolicy: inventory.StatusPolicyAll}.NewClient(tf)
```

Comment:
StatusPolicyAll removed because it's the default?

Reply:
Should we rename it ConfigMapInventoryClient?

Reply:
https://github.com/kubernetes-sigs/cli-utils/pull/661